### DISCUSSION TITLE
2022-08-24 Weekly Meeting 70 Agenda - Avatars and Portals pt. ii

### DISCUSSION THREAD

**Discussion by antpb** [omigroup/omigroup#282] [Weekly Meetings]
Weekly Meeting 70 is happening on Wednesday August 23th, 2022 from 9:00 AM PST(16:00 UTC) - 10:00 AM PST (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).
We regularly use the # omi-meeting-chat channel for text chat. 

## Who we are and why we're here:

The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal. 

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

## Agenda
- Volunteer to take notes? Volunteer to record? 
- Welcome newcomers
- Check in on MSF

### Working Group Updates:
- [OMI Website update](https://github.com/omigroup/website)
- [Scripting group](https://github.com/omigroup/omi-scripting-group)
  - OMI developer hour
- [glTF Extensions Group](https://github.com/omigroup/gltf-extensions)
  - Progress on OMI_ref and authoring extensions
- [Media Group](https://github.com/omigroup/media)
- any others?

### Discussions: 
1) Web-based games and interoperability

![image](https://user-images.githubusercontent.com/8985705/184725567-25383ee2-e435-4f19-9b8c-3294c975dcc5.png)

### Future demo themes: (Vote now in the `#omi-vote` Discord channel!)
- `Wearables`
- `OMI-audio-emitter` exploration and demo 
- `OMI-collider` exploration and demo 
- `Avatars` Pt II
- `Portals` Pt II 
- `Generative and AI tooling`

**Comment by madjin** [comment]
This week we're going to show n' tell demos of interoperable portals and avatars!
![image](https://user-images.githubusercontent.com/32600939/186465947-e747fcf6-c376-4edb-8286-0f377e7b2aa3.png)

**MSF Delegates Meetup**
Got a miro board up to brainstorm OMI x MSF framework to figure out processes, follow-ups, charter, exploratory groups, etc
Categorizing members, brainstorming roles for MSF

- https://github.com/omigroup/MSF-Delegates
- miro link: https://miro.com/welcomeonboard/d2w2cHR0VlY3N2ZKakpoeURaSXJJaWYzTjVmejM0Rnk2RzJiVWx0Z25STU1UOG1oRkRzNkdLejNsOERPMzNGcHwzNDU4NzY0NTMxMzY4NzMwNjIy?share_link_id=814708798368
![image](https://user-images.githubusercontent.com/32600939/186468053-5ff9a176-0c81-427e-b169-e92d312bb117.png)

Lot of overlap between OMI for interop with 3D assets
![image](https://user-images.githubusercontent.com/32600939/186468191-130d7800-cd3a-451c-bc30-1982f6fd051a.png)

Got some discord channels up in the discord
![image](https://user-images.githubusercontent.com/32600939/186467828-28844dcf-6c40-4716-ad30-9f0c1bfbdb1f.png)

**Next media group meetup is September 6th**

UX Research group discussions with Protocol Labs updates:
- New channel made in the discord `#ux-research-pro-temp`
- Bit time constrained at the moment, unsure about leading working group atm
- Working on bunch of metaverse UX research happy to contribute / share soon

Recommend setting up a time for meetup, a working group proposal, lots of interest already

**Comment by futuristudios** [comment]
video recording of conversation available on [ipfs](https://ipfs.io/ipfs/QmaWM3vqyXoKEYvWAgXkSsJUJM5WzHdamBxycFgjEnVx76?filename=OMI_TownHall_2022-08-24%2017-12-20.mp4).

-A
